### DISCUSSION TITLE
What's going on with releases ?

### DISCUSSION THREAD

**Discussion by pleclech** [hasura/graphql-engine#9682] [Question]
Since more or less version v2.20 all features are for Cloud / Enterprise edition only :/ ?

**Comment by m-rgba** [comment]
CE stands for Community Edition in our published images.
There isn't a separate Pro / Cloud / Enterprise image published publicly anymore. 
The features that used to be in Pro / Cloud / Enterprise can be unlocked with an enterprise key (or via trial) and'll take effect when rebooting the container (we did this so user's wouldn't need to change images when upgrading their environments, and so we wouldn't need to maintain, build, and release multiple images).

**Comment by pleclech** [comment]
Thank you:
But is not what i mean , i know the difference between the two versions, my concerns is where are you going ?
Almost all the features are reserved for Cloud/Enterprise since v2.20

**Reply by raajnadar** [reply]
It seems they are focusing more on devs those who pay
